.travis.yml

```diff
@@ -12,22 +12,22 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.6
-      env:
-        - DB=MYSQL
-    - php: 7.0
-      env:
```

Comment:
Happy to lose-out on 5.x, and I appreciate 7.0 is EOL, but SilverStripe itself still works on 7.0 and there's nothing particular about the module that precludes it from running on 7.0 - can it be made to work? (I know how much of a PIA it is to work with Travis btw! :-)

Reply:
FWIW< the README currently stipulates <7.2, which should prob be updated to reflect whatever we decide to do here. Thanks!

Reply:
@phptek True. I have no opinion on which minimum version tbh. We don't and won't develop or run on anything lower than 7.2 from now on as 7.1 hits EOL in a few months.

The composer file should reflect this as well (and is currently at =<7.2).

Do you want me set tests/composer/readme from 5.6 til 7.3? 7.0 to 7.3? Or any other combination?

Reply:
7.0+ please. I can update the docs  myself if you like?

Reply:
@phptek already changed it. Had not read your message